### PR TITLE
Add new owners

### PR DESCRIPTION
**Pull request by Honry**
Add @alexshalamov, @pozdnyakov to owner list for helping test review

### CODE REVIEW

**Comment by tobie** [comment]
Hey, can you please remove me from all of those while you're at it?

Thanks!

**Comment by Honry** [comment]
@tobie, @dontcallmedom, @zqzhang, @Volker-E, if you won't like to be noised by the automatic notification from wpt-bot, I could help remove.

**Comment by Honry** [comment]
@tobie, sure.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/248678402)

# Firefox (nightly) #
Testing web-platform-tests at revision e826a4b8f2126e594af94619b5ecdd5623c9cbc3
Using browser at version BuildID 20170612100241; SourceStamp 27cad9749cddf68e11fdd4e5d73dad84a8f8cf23
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/248678403)

# Sauce (safari) #
Testing web-platform-tests at revision e826a4b8f2126e594af94619b5ecdd5623c9cbc3
Using browser at version 10.0
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/248678409)

# Chrome (unstable) #
Testing web-platform-tests at revision e826a4b8f2126e594af94619b5ecdd5623c9cbc3

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/248678410)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision e826a4b8f2126e594af94619b5ecdd5623c9cbc3

File: generic-sensor/OWNERS [hunk 1]
```diff
@@ -2,3 +2,6 @@
 @dontcallmedom
 @tobie
```
Comment (tobie) [inline generic-sensor/OWNERS]: This one too, please :)

**Review by tobie — APPROVED** [review]

File: generic-sensor/OWNERS (hunk 1 above)
Comment (Honry) [inline generic-sensor/OWNERS]: Done, :)

**Pull request merged**